tests/src/main/java/com/hazelcast/stabilizer/tests/map/MapEvictAndStoreTest.java

```diff
@@ -0,0 +1,102 @@
+package com.hazelcast.stabilizer.tests.map;
+
+import com.hazelcast.config.MapConfig;
+import com.hazelcast.config.MapStoreConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.stabilizer.test.TestContext;
+import com.hazelcast.stabilizer.test.annotations.Run;
+import com.hazelcast.stabilizer.test.annotations.Setup;
+import com.hazelcast.stabilizer.test.annotations.Verify;
+import com.hazelcast.stabilizer.test.utils.AssertTask;
+import com.hazelcast.stabilizer.test.utils.ThreadSpawner;
+import com.hazelcast.stabilizer.tests.map.helpers.MapStoreWithCounter;
+import com.hazelcast.stabilizer.tests.map.helpers.MapStoreWithCounterPerKey;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static com.hazelcast.stabilizer.test.utils.TestUtils.assertTrueEventually;
+import static com.hazelcast.stabilizer.test.utils.TestUtils.isMemberNode;
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertTrue;
+
+/**
+ * Created by dirk on 19/01/15.
+ */
+public class MapEvictAndStoreTest {
+
+    private final static ILogger log = Logger.getLogger(MapEvictAndStoreTest.class);
+
+    public String basename = "MapEvictAndStore";
+    private TestContext testContext;
+    private HazelcastInstance targetInstance;
+
+    private int threadCount = 10;
```

Comment:
Could you please make `threadCount` public, since it is a common property which we can overwrite via the `test.properties` file. Would you also be so kind to group the public and private fields, so we can distinguish properties and internal variables better.